api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -66,19 +66,28 @@ message RBAC {
 
   // Should we do safe-list or block-list style access control?
   enum Action {
-    // The policies grant access to principals. The rest is denied. This is safe-list style
+    // The policies grant access to principals. The rest are denied. This is safe-list style
     // access control. This is the default type.
     ALLOW = 0;
 
-    // The policies deny access to principals. The rest is allowed. This is block-list style
+    // The policies deny access to principals. The rest are allowed. This is block-list style
     // access control.
     DENY = 1;
+
+    // All requests are allowed. The filter sets the `envoy.log` dynamic metadata key to "yes" if
```

Comment:
Should this key be qualified by RBAC filter?

Reply:
This is probably the only comment I have still outstanding.

Reply:
+1. The "envoy" part in "envoy.log" does not provide much value.